src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Android/AndroidCertificatePal.cs

```diff
@@ -85,11 +95,30 @@ private static bool TryReadX509(ReadOnlySpan<byte> rawData, [NotNullWhen(true)]
             return true;
         }
 
+        private static ICertificatePal ReadPkcs12(ReadOnlySpan<byte> rawData, SafePasswordHandle password)
+        {
+            using var reader = new AndroidPkcs12Reader(rawData);
```

Comment:
Please use braces on usings, visible scopes in security/cryptography code make me happy :smile:.

Reply:
Will do.
(First time I decided to try the new syntax :P Not entirely sure how I feel about it, other than that doing it in the middle of the outer scope is definitely unacceptable)

Reply:
I feel like our collective style opinion is that we've shrugged and allowed them in tests, but that no (or at least very little) code in product assemblies uses it... or at least that's been the case for code I've seen across various areas.

Sort of/kind of our style guidelines require them, but there's a lot of wriggle room in the sentence that implies that :smile: